### PR TITLE
[RORO-4843][JT] update banner and content

### PR DESCRIPTION
**Pull request by tianjianfeng**

### CODE REVIEW

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html [hunk 1]
```diff
@@ -62,18 +74,25 @@ <h2 class="govuk-heading-l">@messages("inspection_needed_import.what_next.h2")</
...
                 }
         } else {
                 Html(
                     s"""<h3 class="govuk-heading-m">${messages("inspection_needed_import.inspection.heading")}</h3>
-                       |<p class="govuk-body">${messages("inspection_needed_import_with_other.p1")}</p> <hr aria-hidden="true" /><br/>""".stripMargin)
+                       |<p class="govuk-body">${messages("inspection_needed_import_with_other.p1")}</p> """.stripMargin)
         }
     }
 
+    @govukButton(
```
Comment (tipperba) [inline app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html:89]: You haven't removed the "Check another GMR" hyperlink, have you? ie. you've added in the button, but the hyperlink is still there, I think.

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html [hunk 2]
```diff
@@ -62,18 +72,27 @@ <h2 class="govuk-heading-l">@messages("inspection_needed_export.what_next.h2")</
...
             }
         } else {
             Html(
                 s"""<h3 class="govuk-heading-m">${messages("inspection_needed_export.inspection.heading")}</h3>
-                       |<p class="govuk-body">${messages("inspection_needed_export_with_other.p1")}</p> <hr aria-hidden="true" /><br/>""".stripMargin)
+                       |<p class="govuk-body">${messages("inspection_needed_export_with_other.p1")}</p>""".stripMargin)
         }
     }
 
+    @govukButton(
```
Comment (tipperba) [inline app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html:87]: Same here, I think

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html (hunk 1 above)
Comment (tianjianfeng) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html:89]: removed

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html (hunk 2 above)
Comment (tianjianfeng) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html:87]: removed.

**Review by tipperba — DISMISSED** [review]

**Review by tipperba — APPROVED** [review]

**Review by tipperba — APPROVED** [review]

**Pull request merged**